demisto_sdk/commands/test_content/test_use_case/template_file.py

```diff
@@ -0,0 +1,68 @@
+"""
+{
+    "additional_needed_packs": {
+        "PackOne": "instance_name1",
+        "PackTwo": ""
+    }
+}
+"""
+
+import pytest
+
+from demisto_sdk.commands.common.clients import (
+    XsiamClient,
+    get_client_conf_from_pytest_request,
+    get_client_from_server_type,
+)
+
+# Any additional imports your tests require
+
+
+@pytest.fixture(scope="class")
+def client_conf(request):
+    # Manually parse command-line arguments
+    return get_client_conf_from_pytest_request(request)
+
+
+@pytest.fixture(scope="class")
+def api_client(client_conf):
+    if client_conf:  # Running from external pipeline
+        client_obj = get_client_from_server_type(**client_conf)
+
+    else:  # Running manually using pytest.
+        client_obj = get_client_from_server_type()
+    yield client_obj
+
+
+class TestExample:
+    @classmethod
+    def setup_class(self):
+        """Run once for the class before *all* tests"""
+        pass
+
+    def some_helper_function(self, method):
+        pass
+
+    @classmethod
+    def teardown_class(self):
+        """Run once for the class after all tests"""
+        pass
+
+    # PLAYBOOK X CHECKING VALID alert
+    def test_feature_one_manual_true(self, api_client: XsiamClient):
+        """Test feature one"""
+        a = api_client.list_indicators()
+
+        assert a is not None
```

Comment:
teach them to use comments
```suggestion
        assert a is not None, "list_indicators should not be None"
```